dhcpv4/TUTORIAL.md

```diff
@@ -236,27 +236,16 @@ func main() {
 	if err != nil {
 		log.Fatalln("Error getting address:", err)
 	}
-	client := async.NewClient()
-	client.LocalAddr = &net.UDPAddr{IP: ip, Port: async.DefaultServerPort, Zone: ""}
-	client.RemoteAddr = addr
-
-	err = client.Open(100)
+	client, err := nclient4.New("eth0", nclient4.WithServerAddr(addr))
```

Comment:
Rather than binding to a specific interface (named `eth0`), should this use `NewWithConn` instead?

https://github.com/insomniacslk/dhcp/blob/6a2c8fbdcc1cc23250a45236cd7b968cb43d76e2/dhcpv4/nclient4/client.go#L172-L176

Reply:
This is a markdown file acting as an example. Do you mean we should change the example?

Reply:
🤦 My mistake! I was reviewing this in the mobile app, and I must have lost context of where I was file-wise.